bundles/org.openhab.binding.automower/src/main/java/org/openhab/binding/automower/internal/things/AutomowerHandler.java

```diff
@@ -89,9 +110,24 @@ public AutomowerHandler(Thing thing) {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        if (command instanceof RefreshType) {
+        Optional<AutomowerCommand> commandName = AutomowerCommand.fromChannelUID(channelUID);
```

Comment:
As you don't do anything with the AutomowerCommand in case the command is a refresh I would fetch the command in the else clause of the refresh check...

Reply:
@marcinczeczko can you comment on this?

Reply:
Yep, it's a good idea. Will fix that

Reply:
Good point. Will update